distribution/src/main/resources/VERSIONS.md

```diff
@@ -10,8 +10,8 @@ This file lists the recommended components for this Service Pack. Use it along w
 | Alfresco Content Services | @project.version@ |
 | Alfresco Share | @alfresco.share.version@ |
 | Alfresco Search Services | 1.1.0 | 1.0.0 |
-| Alfresco Desktop Sync Service | 2.1.1 | 2.1.0 |
-| Document Transformation Engine | 2.1.1 | 2.0.x |
+| Alfresco Desktop Sync Service | @alfresco.desktop-sync.version@ | 2.1.0 |
+| Document Transformation Engine | @alfresco.transformation-engine.version@ | 2.0.x |
```

Comment:
Please remove the "also available" versions for DSync and DTE as those versions will not be compatible with 6.0.